src/index.html

```diff
@@ -10,7 +10,56 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class=" search-form"
+    method="get"
+    action="/"
+    data-qa="big"
+    >
+      <label>
+        <div>
+          <img
+          src="images/Search.svg"
+          alt="the icon of a magnifier"
+          class="icon-1"
+          >
+          <input
+          class="input input-big"
```

Comment:
checklist p.2 https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

Reply:
Thank you for the comments. Could you, please, clarify this one. I have specifically used class 'input' for the same attributes, for not to repeat myself twice. So I can't name the 'input-big' just input, because it will just break my code. What should I use instead? like 'input--default' then 'input-small' or what? Am I making myself clear at that point?

Reply:
The problem is that for modifiers we use `--` in the name.
Also, please pay attention to the code formatting. Do as you did above
![image](https://user-images.githubusercontent.com/96409563/199300922-ebc35448-b4a6-4b1d-a7b3-9ddd866611e6.png)
You can read more about BEM syntax here:
https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
https://css-tricks.com/bem-101/